include/mbedtls/sha256.h

```diff
@@ -108,7 +113,9 @@ int mbedtls_sha256_starts_ret( mbedtls_sha256_context *ctx, int is224 );
  *                 SHA-256 checksum calculation.
  *
  * \param ctx      The SHA-256 context.
+ *                 Must not be \c NULL.
  * \param input    The buffer holding the data.
+ *                 Must not be \c NULL if \p ilen is greater than 0.
```

Comment:
Minor: \c 0.

Reply:
Suggestion in line with the other PRs: This must be a readable buffer of length \p ilen Bytes. It may be \c NULL if \p ilen is zero.